plugin.audio.rne/resources/tests/test_api.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Remove tests folder from the submission pls

Reply:
Done as well.